modules/primer-popover/lib/popover.scss

```diff
@@ -164,11 +164,11 @@
   }
 
   &::before {
-    bottom: $spacer-4;
+    bottom: $spacer-3;
   }
 
   &::after {
-    bottom: 21px;
+    bottom: $spacer-3 + 1;
```

Comment:
I'm trying to understand the changes here - is it true that `margin-top` usually takes care of the one-pixel-offset before before/after pseudo elements, but not in this case since these are positioned relative to the `bottom`, not `top`?